packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -101,6 +101,7 @@ export class RichTextEditor extends FoundationElement {
      */
     public boldButtonClick(): void {
         this.tiptapEditor.chain().focus().toggleBold().run();
+        this.focusEditorInFirefox();
```

Comment:
Anything stopping us from writing a unit test to cover this behavior? We already run the tests for nimble-components in both chrome and firefox so if you can come up with a test that failed in FF before but passes now then our infrastructure will validate it.

You might be able to achieve this with a combination of `document.activeElement` (which should be the rich text editor) and `document.activeElement.shadowRoot.activeElement` (which should be the element containing tiptap).

It's possible that Firefox was always reporting the rich text editor as the activeElement, even though it wasn't rendering the cursor. That'd make it hard to write a test that failed in Firefox before this change. However I still think it'd be worthwhile to have testing that verifies the browser is *reporting* the correct activeElement.

And some general feedback: if the PR is only being validated with manual tests then it's helpful to explain the rationale for that in the Testing section of the description.

Reply:
Thanks @jattasNI. I thought of adding a comment for testing this behavior and couldn't find any good way to test the blinking caret as you mentioned below. At least `activeElement` validation will be good.

> It's possible that Firefox was always reporting the rich text editor as the activeElement, even though it wasn't rendering the cursor. That'd make it hard to write a test that failed in Firefox before this change. However I still think it'd be worthwhile to have testing that verifies the browser is reporting the correct activeElement.

Makes sense on the PR description elaboration. Internally aligned with @aagash-ni on the same to pursue it in follow-up PRs.